dot-atom/snippets.cson

```diff
@@ -13,6 +13,8 @@
 #     'prefix': 'log'
 #     'body': 'console.log $1'
 #
+# Each scope can only be declared once.
```

Comment:
Since the text above this line doesn't mention the word "scope", I'm again concerned that this won't be clear enough for some users (they won't know what a "scope" is and what the problem is exactly). For that reason, I think an example would be helpful here, similar to https://github.com/atom/atom/pull/7562/files#r33752061